packages/app-config-writer/src/preview-config/ui5-yaml.ts

```diff
@@ -102,14 +99,32 @@ async function isUi5YamlAlreadyConverted(
  * @param configuration - the preview configuration
  * @returns indicator if the path is an FLP path
  */
-function pathIsFlpPath(path: string | undefined, configuration: PreviewConfig): boolean {
+function isFlpPath(path: string | undefined, configuration: PreviewConfig): boolean {
     if (!path) {
         return false;
     }
-    const isNotRtaEditorPath = configuration.rta?.editors?.every((editor) => editor.path !== path) ?? true;
-    const isNotTestPath = configuration.test?.every((test) => test.path !== path) ?? true;
+    const isRtaEditorPath = configuration.rta?.editors?.some((editor) => editor.path === path) ?? false;
+    return !isRtaEditorPath && !isTestPath(path, configuration);
+}
 
-    return isNotRtaEditorPath && isNotTestPath;
+/**
+ * Check if the path is a test path.
+ * 1) path matches pattern '**.qunit.html'
+ * 2) path is being used as test configuration path in yaml configuration.
+ *
+ * @param path - the path
+ * @param configuration - the preview configuration
+ * @returns indicator if the path is a test path
+ */
+export function isTestPath(path: string | undefined, configuration?: PreviewConfig): boolean {
+    if (!path) {
+        return false;
+    }
+    //todo: how to ensure we don't mistake other test scripts for preview scripts?
```

Comment:
@tobiasqueck: do you have a better idea to identify test (runner) files than eiter filename contains `.qunit.html` or `path` is being used as `test.path`  in yaml config of preview-middleware?